src/resources/views/base/widgets/card.blade.php

```diff
@@ -4,7 +4,7 @@
 @endphp
 
 @includeWhen(!empty($widget['wrapper']), 'backpack::widgets.inc.wrapper_start')
-	<div class="{{ $widget['class'] ?? 'card' }}">
```

Comment:
Same here. I think in general... if on this branch the change is to add a helper class for margin/padding/color/etc... it means that should have been done in the CSS file instead, using a specific `backpack-*` class.